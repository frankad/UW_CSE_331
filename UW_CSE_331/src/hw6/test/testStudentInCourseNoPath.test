# Find a path with multiple hops from one student to an other students 
# using studentInCourse dataset, a case no path beween two students.

# build graph
LoadGraph studentInCourse studentInCourse.tsv

# test the findPathBFS
FindPath studentInCourse Jeff Emily