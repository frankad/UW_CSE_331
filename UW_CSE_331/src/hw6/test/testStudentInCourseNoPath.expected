# Find a path with multiple hops from one student to an other students 
# using studentInCourse dataset, a case no path beween two students.

# build graph
loaded graph studentInCourse

# test the findPathBFS
path from Jeff to Emily:
no path found